tests/models/gemma/test_modeling_gemma.py

```diff
@@ -851,27 +803,10 @@ def test_compile_static_cache(self):
             self.skipTest(reason="This test requires torch >= 2.3 to run.")
 
         NUM_TOKENS_TO_GENERATE = 40
-        # Note on `EXPECTED_TEXT_COMPLETION`'s diff: the current value matches the original test if the original test
-        # was changed to have a cache of 53 tokens (as opposed to 4096), on Ampere GPUs.
```

Comment:
This comment was originally written by me in the equivalent llama test. It was probably copied around -- it does not apply here :)